modules/nosql/manager_redis.go

```diff
@@ -59,8 +60,84 @@ func (m *Manager) GetRedisClient(connection string) redis.UniversalClient {
 		name: []string{connection, uri.String()},
 	}
 
+	opts := getRedisOptions(uri)
+	tlsConfig := getRedisTLSOptions(uri)
+
+	clientName := uri.Query()["clientname"]
+
+	if clientName != nil && len(clientName) > 0 {
```

Comment:
```suggestion
	if len(clientName) > 0 {
```

In Golang, the `clientName != nil` is not needed when doing `len`.

And if you only need one value, you can use `uri.Query().Get()` directly, without these checks.

Reply:
Thanks, yeah I think in this case we can use the getter since we don't need the distinction on `nil`. This was originally just pulled out of the switch statement that existed before where `nil` needed to be handled.

I'll swap this one around!